### ISSUE TITLE
Keyboard Navigation in Dropdown menu does not work

### ISSUE THREAD

**Issue by narg95** [angular-ui/bootstrap#1931]
Hello,

The following example uses bootstrap original libraries http://jsfiddle.net/miksh/xEBp4/ .  Navigation in drop down menu is possible with the UP/DOWN keys.  Nevertheless I could not make it work with Angular-bootstrap libraries see http://plnkr.co/edit/rv0UT8UGOaH0p6jkbJJd?p=preview 
Am I missing something ?

Thanks,
Nestor
